tests/framework/installer/ceph_installer.go

```diff
@@ -753,13 +753,14 @@ spec:
                       #
                       udevadm trigger || true
                       vgimport -a || true
+                      PVS=$(pvs --noheadings --separator=' ' -o pv_name,vg_name | awk '/ceph-/ {print $1}')
                       # Wipe VGs
-                      for vg in $(vgs --noheadings --readonly --separator=' ' -o vg_name); do
+                      for vg in $(vgs --noheadings --readonly --separator=' ' -o vg_name | awk '/ceph-/ {print}'); do
```

Comment:
Aren't missing a column name from the print? Also, I think @BlaineEXE suggestion is probably better.

Reply:
Oops, sorry. And I'll adapt Blaine's suggestion.